birt-osgi/build.gradle

```diff
@@ -28,24 +28,15 @@ task prepareBirtOsgi(description: 'Prepare BIRT OSGI') {
       from zipTree(originalFile)
       into osgiFolder
     }
-
-    println("Download the OSGI compatibility Jar")
-    new URL("https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.osgi.compatibility.state/1.2.500/org.eclipse.osgi.compatibility.state-1.2.500.jar").withInputStream {
-      compatibilityJar << it
-    }
-
-    println("Update file 'bundles.info'")
-    file("${platformFolder}/configuration/org.eclipse.equinox.simpleconfigurator/bundles.info")
-      .append("org.eclipse.osgi.compatibility.state,1.2.500.v20210730-0750,plugins/org.eclipse.osgi.compatibility.state-1.2.500.jar,4,false")
```

Comment:
Why is this dropped?

Reply:
This is because the compatibility JAR is included in v4.16 and the bundle file has the info this JAR.

Reply:
Goodoh, I say merge on then. :tada: